### PR TITLE
Update to 2.2.0

### PR DESCRIPTION
**Pull request by jschueller**
@conda-forge-admin please rerender

### CODE REVIEW

**Comment by jschueller** [comment]
@conda-forge/tensorflow-estimator I think this is ready to go

**Comment by jschueller** [comment]
@conda-forge-admin please rerender

**Comment by jschueller** [comment]
ping @conda-forge/tensorflow-estimator

File: recipe/meta.yaml [hunk 1]
```diff
@@ -30,18 +29,14 @@ requirements:
...
     - wrapt >=1.11.1
     - grpcio >=1.8.6
-    - enum34 >=1.1.6              # [py<34]
-    - backports.weakref >=1.0rc1  # [py<34]
-    - mock >=2.0.0                # [py2k]
-
 
 test:
   imports:
     - numpy  # NB: can't import tensorflow on CI due to glibc compat issues.
```
Comment (xhochy) [inline recipe/meta.yaml]: Is this still the case?

Comment (jschueller) [reply recipe/meta.yaml]: Yes, it still fails

**Review by xhochy — APPROVED** [review]

**Pull request merged**